### ISSUE TITLE
Traceback if all genomes are filtered

### ISSUE THREAD

**Issue by Askars** [Ecogenomics/GTDBTk#108]
Hi all,
I get a traceback in the following circumstances.

Running v0.2.2. If all genomes in a domain are filtered by the MSA column filter, an unhandled exception will be thrown. Usually affects archaea as they usual have the least number of genomes and may be quite incomplete.

Step to reproduce: 

Data here:
https://drive.google.com/file/d/1fARFlmavIg1oEo-jhcacN7IU6W9STm5c/view
```
gtdbtk classify_wf --cpus 10 -x fa --genome_dir bins_test --out_dir gtdbtk_test
```

```
[2019-03-29 11:44:35] INFO: GTDB-Tk v0.2.2
[2019-03-29 11:44:35] INFO: gtdbtk classify_wf --cpus 10 -x fa --genome_dir bins_test/ --out_dir gtdbtk_test2
[2019-03-29 11:44:35] WARNING: Results are still being validated and taxonomic assignments may be incorrect! Use at your own risk!
[2019-03-29 11:44:35] INFO: Identifying markers in 2 genomes with 10 threads.
[2019-03-29 11:44:35] INFO: Running Prodigal to identify genes.
==> Finished processing 2 of 2 (100.0%) genomes.
[2019-03-29 11:44:44] INFO: Identifying TIGRFAM protein families.
==> Finished processing 2 of 2 (100.0%) genomes.
[2019-03-29 11:44:47] INFO: Identifying Pfam protein families.
==> Finished processing 2 of 2 (100.0%) genomes.
[2019-03-29 11:44:48] INFO: Done.
[2019-03-29 11:44:48] INFO: Aligning markers in 2 genomes with 10 threads.
[2019-03-29 11:44:48] INFO: Processing 1 genomes identified as bacterial.
[2019-03-29 11:44:49] INFO: Read concatenated alignment for 17413 GTDB genomes.
[2019-03-29 11:44:53] INFO: Masking columns of multiple sequence alignment using canonical mask.
[2019-03-29 11:45:31] INFO: Masked alignment from 41155 to 5035 AA.
[2019-03-29 11:45:31] INFO: 0 user genomes have amino acids in <10.0% of columns in filtered MSA.
[2019-03-29 11:45:31] INFO: Creating concatenated alignment for 17414 GTDB and user genomes.
[2019-03-29 11:45:31] INFO: Creating concatenated alignment for 1 user genomes.
[2019-03-29 11:45:31] INFO: Processing 1 genomes identified as archaeal.
[2019-03-29 11:45:31] INFO: Read concatenated alignment for 845 GTDB genomes.
[2019-03-29 11:45:32] INFO: Masking columns of multiple sequence alignment using canonical mask.
[2019-03-29 11:45:34] INFO: Masked alignment from 32675 to 5024 AA.
[2019-03-29 11:45:34] INFO: 1 user genomes have amino acids in <10.0% of columns in filtered MSA.
[2019-03-29 11:45:34] INFO: Creating concatenated alignment for 845 GTDB and user genomes.
[2019-03-29 11:45:34] INFO: Creating concatenated alignment for 0 user genomes.
[2019-03-29 11:45:34] INFO: Done.
Traceback (most recent call last):
  File "/sw/py27_anaconda_modules/gtdbtk/0.2.2/lib/python2.7/site-packages/gtdbtk/biolib_lite/seq_io.py", line 163, in read_fasta_seq
    yield seq_id, ''.join(seq).replace(' ', '')
TypeError: can only join an iterable


[Error] Failed to process sequence file: gtdbtk_test2/gtdbtk.ar122.user_msa.fasta

Controlled exit resulting from an unrecoverable error or warning.

```

**Comment by pchaumeil** [comment]
Hi Adam,
Thanks for your feedback !
This bug should be fix in d3eaa4c578658868d8c2ea1ea20a2e49547b98ea and will be available in the next release.

Cheers,
Pierre

**Comment by Askars** [comment]
Woo thanks Pierre.

Cheers,
Adam
